projects/demo/src/modules/experimental/badge/badge.template.html

```diff
@@ -0,0 +1,164 @@
+                >
+                    <tui-svg src="tuiIconCheck"></tui-svg>
+                </tui-badge>
+
+                <tui-badge
+                    *ngIf="contentType === 'empty'"
+                    [appearance]="appearance"
+                    [hoverable]="hoverable"
+                    [size]="size"
+                    [value]="values[value]"
+                ></tui-badge>
+
+                <tui-badge
+                    *ngIf="contentType === 'image'"
+                    [appearance]="appearance"
+                    [hoverable]="hoverable"
+                    [size]="size"
+                >
+                    <img
+                        alt="market"
+                        src="assets/images/avatar.jpg"
+                    />
+                </tui-badge>
+            </p>
+        </tui-doc-demo>
+        <label
+            tuiLabel="Content type"
+            class="tui-space_vertical-3"
+        >
+            <tui-radio-list
+                orientation="horizontal"
+                [items]="contentTypeVariants"
+                [(ngModel)]="contentType"
+            ></tui-radio-list>
+        </label>
+        <tui-doc-documentation>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="size"
+                documentationPropertyType="TuiSizeL"
+                [documentationPropertyValues]="sizeVariants"
+                [(documentationPropertyValue)]="size"
+            >
+                Size
+            </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="appearance"
+                documentationPropertyType="TuiStatus"
+                [documentationPropertyValues]="appearanceVariants"
+                [(documentationPropertyValue)]="appearance"
+            >
+                Appearance
+            </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="value"
+                documentationPropertyType="string | number"
+                [documentationPropertyValues]="valueVariants"
+                [(documentationPropertyValue)]="value"
+            >
+                Value. If value is a number and is more than 99, it will be replaced with &laquo;99+&raquo;
+            </ng-template>
+            <ng-template
+                documentationPropertyMode="input"
+                documentationPropertyName="hoverable"
+                documentationPropertyType="boolean"
+                [(documentationPropertyValue)]="hoverable"
+            >
+                Enable hovered state
+            </ng-template>
+        </tui-doc-documentation>
+    </ng-template>
+
+    <ng-template pageTab>
+        <ol class="b-demo-steps">
+            <li>
+                <p>
+                    Import
+                    <code>TuiBadgeModule</code>
```

Comment:
How about we start to write just "Import into a module"? That way we wouldn't have to edit this part every time we create a new page :D